packages/client/src/constants.ts

```diff
@@ -11,5 +11,18 @@ export const appConfig = {
   sentryDsn: process.env.REACT_APP_SENTRY_DSN,
 };
 
+export const builtInActivities = [
```

Comment:
@brackenish hey sorry man. this is what i was talking about.
I just thought that maybe this didnt belong in constants, since it wont be used in other places aside from activities. I think this file should be for things that are global available.

Please sir, if ya dont mind, can you move this to be colocated with activities. 

or put up your best argument against this.

/giphy  tmnt turtle power

Reply:
Fixed!